runtime/Dockerfile

```diff
@@ -41,6 +41,23 @@ RUN echo "Installing base packages..." \
     && xargs -a /home/${RUNTIME_USER}/apt.txt apt-get install -y \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /home/${RUNTIME_USER}/apt.txt
+
+
+ENV APT_INSTALL="apt-get install -y --no-install-recommends"
+RUN apt-get update && \
+    cd /tmp && wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB && \
+    apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2019.PUB && \
+    sh -c 'echo deb https://apt.repos.intel.com/mkl all main > /etc/apt/sources.list.d/intel-mkl.list' && \
+    apt-get update && DEBIAN_FRONTEND=noninteractive $APT_INSTALL intel-mkl-64bit-2020.4-912
+
+
+# Runtime dependencies for the project
+ENV LD_LIBRARY_PATH="/opt/intel/mkl/lib/intel64/:$LD_LIBRARY_PATH"
+ENV MKLROOT="/opt/intel/mkl"
+ENV KENLM_ROOT=/opt/kenlm
```

Comment:
Hi @jayqi,
The algorithms we're deploying for the challenge are part of a more extensive app we're working on that's built on C++ and compiled via Bazel with high hermicity; because of this the app doesn't link properly (in runtime) to the MKL package if deployed with conda. 
For us it would be great if we could have the Intel-MKL package installed during image creation. I understand thought that this is not the standard for this particular environment; if it's not possible to have it that way we'll work on redeploying our app to sort it out.
Please let me know with any decision so we can proceed accordingly.
Thank you!!

Reply:
Hi @icirauqui,

I think we can potentially consider installing a system MKL to be a backup option. Ideally, we can get your app to link to the conda MKL. Do you have a clear understanding of the requirements for making that happen, or is it unknown? Would being able to set the environment variables with the right paths to the conda MKL be enough? 

Additionally, it may be more expedient to separate the installation of eigen into another PR. We can get that merged in separately more quickly.

Reply:
Hi @jayqi,

I've created a separate PR for Eigen and gnupg2.

Regarding linking the app to the Conda MKL, at the moment, the process for doing it is unknown, just linking it to the apt installation was a delicate process at the beginning. Yes, setting the proper environment variables pointing to Conda's MKL might work, but I'm afraid that we're still trying to validate it.

Reply:
@icirauqui You made find the [`install.sh` script](https://www.drivendata.org/competitions/105/nist-federated-learning-2-financial-crime-federated/page/587/#optional-install-sh-script) support that was added yesterday useful in setting up linking in MKL.

If you aren't able to get the Conda MKL working, then we can come back to the apt installation as a backup option.

Reply:
Hi @icirauqui, just checking in on whether you've been able to get Conda's MKL to work with your code, or if we need to look at this further.